apps/common/drf/parsers/base.py

```diff
@@ -126,7 +126,7 @@ def parse_value(self, field, value):
                 value = self.id_name_to_obj(value)
         elif isinstance(field, LabeledChoiceField):
             value = self.id_name_to_obj(value)
-            if isinstance(value, dict) and value.get('pk'):
+            if isinstance(value, dict) and 'pk' in value:
                 value = value.get('pk')
         elif isinstance(field, serializers.ListSerializer):
             value = [self.parse_value(field.child, v) for v in value]
```

Comment:
There are no notable differences between the code you provided and a typical representation of `parse_value` function from Django REST Framework on 2023-09-01.

However, note that these functions often implement different behaviors or use different strategies to determine how certain data should be parsed based on its type:

- The original version has slightly better formatting (i.e., indentation).

So it is recommended to keep an eye out for changes and updates in the API specification as newer versions could introduce significant enhancements. It may be worth checking with someone who can verify whether there’s been anything particularly significant updated since then (as of September 1st 2021).